internal/services/logz/logz_monitor_resource_test.go

```diff
@@ -240,5 +240,6 @@ resource "azurerm_logz_monitor" "test" {
 }
 
 func getLogzInstanceName(randomInteger int) string {
+	// Name started with `liftr_test_only_` will be regarded as a test resource and cleared by the back end
```

Comment:
Hi @tombuildsstuff, I have removed this function and set the resource name start with `acctest-`.
At first, I used the prefix of `liftr_test_only_` because when we create a new monitor/sub account resource with an unused email address, a new Logz account with this email address will be created in https://logz.io. After the monitor/sub account resource is deleted in Azure Portal, this Logz account will not be deleted in https://logz.io. If we use random email addresses in test cases every time, whenever a test case is run, a new Logz account will be created in https://logz.io, which will cause multiple test email addresses created in https://logz.io after a few days. But if the resource name starts with `liftr_test_only_`, the Azure Logz Service will notify https://logz.io that this email address is for test only and could be ignored safely. 
Currently, to prevent the test email data pollution in https://logz.io and still use `acctest-` in our test cases, I give each test case a fixed email address. Kindly help confirm the latest code again. Thanks.